runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/MetricsContainerImpl.java

```diff
@@ -146,8 +148,19 @@ public MetricUpdates getUpdates() {
 
     for (MetricUpdate<Long> mu : mus.counterUpdates()) {
```

Comment:
mu, as well as mus are hard to read. Use monitoringInfo and monitoringInfos.
e -> entry. (can be mixed with error, or other abbreviations)